lib/assessment.md

```diff
@@ -163,5 +181,5 @@ Consider the following media queries. How wide would the screen need to be for e
 ```
 
 <!-- your answer starts here -->
-
+at 150 pixel screen. it will have the width of 100px.  so no stretching.
```

Comment:
Sorry, that's not correct. This question was asking what screen width would be necessary for `.stretch` elements to have a width of 150px.